docs/developers/README.md

```diff
@@ -1,6 +1,22 @@
 # Developers
+
 Developer features allow extending and enhancing Chart.js in many different ways.
 
+# Latest documentation
+
+The latest version of the documentation, including unreleased features, is always available at the [documentation master](http://www.chartjs.org/docs/master/).
+
+# Testing development releases
+
+Tools like [JSFiddle](http://jsfiddle.net) and [CodePen](http://codepen.io) allow quick prototying and validation of Chart.js functionalities. One common problem is that this requires the built version of the Chart.js library.
+
+Latest builds are available for testing at:
+
+ - http://chartjs.org/dist/master/Chart.min.js
+ - http://chartjs.org/dist/master/Chart.bundle.min.js
+
+*Note*: These development builds are only available via HTTP. In order to include them in JSFiddle or CodePen, these need to be accessed via HTTP, too. The development build MUST not be used for production purposes or as replacement for CDN.
```

Comment:
In addition to the MUST (all uppercase), I would put in bold the whole phrase on another line:

> *Note*: These development builds are only available via HTTP. In order to 

> **WARNING: the development .... for CDN**".

```markdown
> *Note*: These development builds are only ... via HTTP, too.

> **WARNING: The development .... for CDN**".
```